content/soarian-financials/stu3/individuals/patient.md

```diff
@@ -0,0 +1,638 @@
+---
+title: Patient | Release 3 (STU) API
+---
+
+# Patient
+
+* TOC
+{:toc}
+
+## Overview
+The Patient resource provides general demographic information about a person receiving health care services from a specific organization. Common demographic fields include patient id, patient name, gender, date of birth, address, phone, primary language and marital status. Soarian Finanicals is a patient centric application: thus, many of the other resources will include the patient id in their queries. A person receiving care from multiple organizations may have data available in multiple patient resources in multiple FHIR servers.
+
+The following fields are returned if valued:
+
+   * [Patient name](http://hl7.org/fhir/patient-definitions.html#Patient.name){:target="_blank"}
+   * [Patient id](http://hl7.org/fhir/patient-definitions.html#Patient.identifier){:target="_blank"}
+   * [Extensions including birth time, preferred telecom, bad address indicator](http://hl7.org/fhir/extensibility-registry.html)
+   * [Contact information (includes phone,faxand email)](http://hl7.org/fhir/datatypes.html#ContactPoint){:target="_blank"}
+   * [Gender (administrative)](http://hl7.org/fhir/patient-definitions.html#Patient.gender){:target="_blank"}
+   * [Date of Birth](http://hl7.org/fhir/patient-definitions.html#Patient.birthDate){:target="_blank"}
+   * [Deceased](http://hl7.org/fhir/patient-definitions.html#Patient.deceased_x_){:target="_blank"}
+   * [Address](http://hl7.org/fhir/patient-definitions.html#Patient.contact.address){:target="_blank"}
+   * [Communication (preferred language)](http://hl7.org/fhir/patient-definitions.html#Patient.communication){:target="_blank"}
+   * [Marital status](http://hl7.org/fhir/patient-definitions.html#Patient.maritalStatus){:target="_blank"}
+   * [Primary Care Providers](http://hl7.org/fhir/patient-definitions.html#Patient.generalPractitioner){:target="_blank"}
+ 
+
+## Extensions
+* [Time of day of birth]
+* [Patient level user data](#patient-user-data)
+* [Bad address indicator](http://hl7.org/fhir/v3/AddressUse/cs.html){:target="_blank"}
+* [Preferred telecom](http://hl7.org/fhir/extension-iso21090-preferred.html){:target="_blank"}
+
+## Search
+
+Search for Patients that meet supplied query parameters:
+
+    GET /Patient/<Soarian Finanicals Person Number>
```

Comment:
this request is a read, not a search. either the title needs to change or the example needs to use query parameters